README.md

```diff
@@ -13,6 +13,25 @@ polluting the markup that is delivered to the end user.
 ember install ember-test-selectors
 ```
 
+## Configuartion
+To modify the default configuration, place a block called `ember-test-selectors`
+in your `ember-cli-build.js` file.
+
+### Options
+
+`envsToStrip`
+Defines the environments in which you want the test selectors to be removed.
+By default, selectors are only removed in the `production` environment. You
+might also want to remove them in other staging environments for testing.
+
+```javascript
+var app = new EmberApp({
+  'ember-test-selectors': {
+    envsToStrip: ['production', 'staging']
+  }
+});
+```
+
```

Comment:
I think this whole paragraph on configuration should be below the paragraphs explaining the motivation for the addon as this will not be as important for most people, especially not when looking at the addon for the first time.